packages/lab/src/Flow/stories/BaseHook/BaseHook.tsx

```diff
@@ -0,0 +1,158 @@
+import { useState } from "react";
+import { css } from "@emotion/css";
+import {
+  HvButton,
+  HvDialog,
+  HvDialogActions,
+  HvDialogContent,
+  HvDialogTitle,
+} from "@hitachivantara/uikit-react-core";
+import {
+  Edit,
+  Favorite,
+  Flag,
+  Search,
+} from "@hitachivantara/uikit-react-icons";
+import {
+  HvFlowInstance,
+  HvFlowNode,
+  HvFlowNodeFC,
+  HvFlowNodeProps,
+  useFlowNode,
+} from "@hitachivantara/uikit-react-lab";
+
+type Node = ReturnType<HvFlowInstance["getNode"]>;
+
+interface AssetData {
+  asset?: string;
+}
+
+const classes = {
+  container: css({
+    width: "40%",
+    minHeight: 200,
+  }),
+  outputLabel: css({
+    display: "flex",
+    alignItems: "center",
+    gap: 2,
+  }),
+};
+
+export const BaseHook: HvFlowNodeFC<AssetData> = (props) => {
```

Comment:
Is this file being used? I couldn't find where.
Also, it has a confusing name. It's called `BaseHook` but `index.tsx` doesn't use it